src/site/stages/build/plugins/check-cms-urls.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable no-console */
+const ignoredPages = new Set(['drupal/test/index.html']);
+
+function checkForCMSUrls(BUILD_OPTIONS) {
+  return (files, metalsmith, done) => {
+    const filesWithBadUrls = [];
+    for (const fileName of Object.keys(files)) {
+      const file = files[fileName];
+      if (file.isDrupalPage && !ignoredPages.has(fileName)) {
+        const contents = file.contents.toString();
+        if (
+          contents.includes('cms.va.gov') ||
+          contents.includes('va.agile6.com')
```

Comment:
Is it worth trying to import the addresses from `constants/drupals.js` and using those here?

Reply:
I figured this was a little simpler and also avoids us missing things because of aliases (e.g. there's stg.va.agile6.com and staging.va.agile6.com).